src/content/reference/SDKs/ios.md

```diff
@@ -858,10 +858,82 @@ List of function names exposed by device. Functions are only available after `re
 
 Dictionary of exposed variables on device with their respective types. Variables are only available after `refresh` is called on the device instance returned by `getDevices` call.
 
-  `@property (strong, nonatomic, readonly) NSString *version`
+  `@property (strong, nonatomic, readonly) NSString *systemFirmwareVersion`
 
 Device firmware version string
 
+  `@property (strong, nonatomic, nullable, readonly) NSDate *lastHeard`
+
+Date when the device was last heard.
+
+  `@property (strong, nonatomic, nullable, readonly) NSString *appHash`
+
+App hash received from system event after flashing a new different user application. Please note that if you flashe same app, the app hash event won't trigger.
+
+  `@property (strong, nonatomic, nullable, readonly) NSString *networkId`
+
+Mesh network ID if device belongs to a mesh network.
+
+  `@property (nonatomic, readonly) ParticleDeviceNetworkRole networkRole`
+
+Mesh network role if device belongs to a mesh network (gateway or a node).
+
+  `@property (nonatomic, readonly) ParticleDeviceNetworkRoleState networkRoleState`
+
+Mesh network state if device belongs to a mesh network. State can be pending if device is waiting for the role change confirmation.
+
+  `@property (nonatomic, readonly) BOOL isFlashing`
+
+Is device currently being flashed? Best effort - May not accurate reflect true state.
+
+  `@property (strong, nonatomic, nullable, readonly) NSString *lastIPAdress`
+
+Last know IP address of the device.
```

Comment:
Should be "last known"

Reply:
Later commits remove this section completely.